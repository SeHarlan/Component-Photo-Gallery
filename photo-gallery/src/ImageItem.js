import React, { Component } from 'react';

export default class ImageItem extends Component {
    render() {
        return <li className='item'>
            <em>category: {this.props.item.keyword}</em>
            <h2>{this.props.item.title}</h2>
            <img src={this.props.item.url} alt={this.props.item.description}/>
            <p>horns: {this.props.item.horns}</p>
        </li>
    }
}