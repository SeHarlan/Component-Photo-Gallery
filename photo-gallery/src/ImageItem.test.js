import React from "react";
import renderer from "react-test-renderer";
import ImageItem from './ImageItem.js';

import item from './data.js';

test('renders ImageList.js correctly', () => {
  const tree = renderer
    .create(<ImageItem item={item}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
