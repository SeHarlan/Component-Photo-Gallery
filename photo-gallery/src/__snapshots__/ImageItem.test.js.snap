// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ImageList.js correctly 1`] = `
<li
  className="item"
>
  <em>
    category: 
  </em>
  <h2 />
  <img />
  <p>
    horns: 
  </p>
</li>
`;
